### PR TITLE
Create customer url link in urls.py

### PR DESCRIPTION
**Pull request by mccordgh**
Created a url reference for bangazon_api app in main bangazon project urls.py

## Related Issue
 #51 

## Motivation and Context
This was required because there was no link to our bangazon_api Customer view.

## How Has This Been Tested?
ran the manage.py server, surfed on over to localhost:8000/customers, and received a lovely view!

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: bangazon/bangazon/urls.py [hunk 1]
```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
-from django.contrib import admin
+
```
Comment (asimonia) [inline bangazon/bangazon/urls.py:16]: I propose that we leave the admin panel inside for testing purposes
```from django.contrib import admin```

A lot of good stuff here.

File: bangazon/bangazon/urls.py [hunk 2]
```diff
@@ -13,9 +13,15 @@
...
+from django.conf.urls import url, include
+from rest_framework import routers
+from bangazon_api import views
+
+router = routers.DefaultRouter()
+router.register(r'customers', views.CustomerViewSet)
 
 urlpatterns = [
```
Comment (asimonia) [inline bangazon/bangazon/urls.py:25]: Add in ```url(r'^admin/', admin.site.urls),``` as well 👍

**Review by asimonia — CHANGES_REQUESTED** [review]
Please review

**Review by asimonia — COMMENTED** [review]
Add in ```url(r'^admin/', admin.site.urls),``` as well 👍

**Review by rtwhitfield84 — APPROVED** [review]
nice 🎃 🌁

**Review by asimonia — APPROVED** [review]

**Pull request merged**